numpy/testing/utils.py

```diff
@@ -44,6 +44,28 @@ class KnownFailureException(Exception):
 verbose = 0
 
 
+def import_nose():
+    """ Import nose only when needed.
+    """
+    fine_nose = True
```

Comment:
Will fix, it is copy paste. Should maybe check whether it is still necessary, I think it was something about circular imports later on.